utils/src/repl/command.rs

```diff
@@ -131,13 +154,63 @@ pub fn parse_browse() -> impl Fn(Span) -> IResult<Span, Command, FileError<Span>
   }
 }
 
+pub fn parse_multiaddr(
+  from: Span,
+) -> IResult<Span, Reference, ParseError<Span>> {
+  let (i, s) = take_till1(|x| char::is_whitespace(x))(from)?;
+  let s: String = String::from(s.fragment().to_owned());
+  match Multiaddr::try_from(s) {
+    Ok(addr) => Ok((i, Reference::Multiaddr(addr))),
+    Err(e) => {
+      Err(Err::Error(ParseError::new(from, ParseErrorKind::InvalidSymbol(e.to_string()))))
+    }
+  }
+}
+
+pub fn parse_cid_reference(from: Span) -> IResult<Span, Reference, ParseError<Span>>
+{
+  let (i, cid) = parse_link(from)?;
+  Ok((i, Reference::Cid(cid)))
+}
+
+pub fn parse_name_reference(from: Span) -> IResult<Span, Reference, ParseError<Span>>
+{
+  let (i, name) = parse_name(from)?;
+  Ok((i, Reference::FileName(name)))
+}
+
+/// Parse the :load command
 pub fn parse_load() -> impl Fn(Span) -> IResult<Span, Command, FileError<Span>>
 {
   move |i: Span| {
     let (i, _) = alt((tag(":load"), tag(":l")))(i)?;
     let (i, _) = parse_space1(i).map_err(error::convert)?;
-    let (i, name) = parse_name(i).map_err(error::convert)?;
-    Ok((i, Command::Load(name)))
+    let (i, reference) = alt((parse_multiaddr, parse_name_reference, parse_cid_reference))(i).map_err(error::convert)?;
+    Ok((i, Command::Load(reference)))
+  }
+}
+
+pub fn parse_word(from: Span) -> IResult<Span, String, ParseError<Span>> {
```

Comment:
You can use `parse_name` from `core::parse::term` here to get the other restrictions on what can be a valid symbol. `parse_name` excludes the yatima reserved words (like `lambda` or `Type`) though and while that's not a problem currently, we may want to refactor with a `reserved_keywords: bool` param, or an inner function so we can have consistent symbolic restrictions and erroring